### PR TITLE
[doc] Improve `hello.rst`

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related issue = #941 

I will try to update the Chinese version of `hello.rst` as well to learn about the Chinese translation process. (I'm setting up environment according to https://github.com/taichi-dev/taichi-docs-zh-cn/blob/master/README.md.)

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: docs/hello.rst [hunk 1]
```diff
@@ -17,64 +17,65 @@ Now you are ready to run the Taichi code below (``python3 fractal.py``) to compu
 
 .. code-block:: python
 
-  # fractal.py
+    # fractal.py
 
-  import taichi as ti
```
Comment (archibate) [inline docs/hello.rst:22]: Why remove this?

File: docs/hello.rst [hunk 2]
```diff
@@ -140,84 +141,108 @@ For those who came from the world of CUDA, ``ti.func`` corresponds to ``__device
...
-                break # OK
+                break # OK!
+            break # Error!
 
 
 Interacting with Python
 ------------------------
 
-Everything outside Taichi-scope (``ti.func`` and ``ti.kernel``) is simply Python. You can use your favorite Python packages (e.g. ``numpy``, ``pytorch``, ``matplotlib``) with Taichi.
+Everything outside Taichi-scopes (``ti.func`` and ``ti.kernel``) is simply Python. You can use your favorite Python packages (e.g. ``numpy``, ``pytorch``, ``matplotlib``) with Taichi.
```
Comment (archibate) [inline docs/hello.rst:245]:
```suggestion
Everything outside Taichi-scope (``ti.func`` and ``ti.kernel``) is simply Python. You can use your favorite Python packages (e.g. ``numpy``, ``pytorch``, ``matplotlib``) with Taichi.
```

File: docs/hello.rst [hunk 3]
```diff
@@ -140,84 +141,108 @@ For those who came from the world of CUDA, ``ti.func`` corresponds to ``__device
...
 
       @ti.kernel
       def foo():
         for i in x:
-            for j in y:
+            for j in range(10):
                 ...
-                break # OK
+                break # OK!
+            break # Error!
```
Comment (archibate) [inline docs/hello.rst:239]:
```suggestion
```

File: docs/hello.rst [hunk 4]
```diff
@@ -140,84 +141,108 @@ For those who came from the world of CUDA, ``ti.func`` corresponds to ``__device
...
+                    ...
 
-.. note::
-    ``break`` is not supported in **outermost (parallelized)** loops:
+
+
+
+.. warning::
+
+    ``break`` **is not supported in parallel loops:**
```
Comment (archibate) [inline docs/hello.rst:219]:
```suggestion
    ``break`` **is not supported in parallel loops**:
```

**Comment by archibate** [comment]
> (I'm setting up environment according to https://github.com/taichi-dev/taichi-docs-zh-cn/blob/master/README.md.)

Thanks, if you find any problem, feel free to ask me how-to-work.

File: docs/hello.rst (hunk 1 above)
Comment (yuanming-hu) [reply docs/hello.rst:22]: Nice catch. I'll add it back.

File: docs/hello.rst (hunk 2 above)
Comment (yuanming-hu) [reply docs/hello.rst:245]: I'd actually stick to plural forms here since we may have multiple `ti.kernel` and each is a Taichi-scope. WDYT?

**Pull request merged**